_locales/en/messages.json

```diff
@@ -1622,6 +1616,12 @@
     "pidTuningRatesSuperExpoHelp": {
         "message": "This setting controls the feature 'SUPEREXPO_RATES'"
     },
+    "pidTuningVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "pidTuningVbatPidCompensationHelp": {
+        "message": "Increases the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight. The amount of compensation that is applied is calculated from Maximum Cell Voltage above, so make this is set to something appropriate."
+    },
```

Comment:
Are you missing a "sure" here? "...so make **sure** this is set to something appropriate." ?